tests/OBO.NET.CodeGeneration.Tests/References/ReferenceSourceFile.fs

```diff
@@ -0,0 +1,11 @@
+﻿namespace ARCTokenization.StructuralOntology
```

Comment:
Why are you not testing the whole ontology? this is just 3 terms, while your reference obo file contains a lot more.